src/Page/Donate/index.js

```diff
@@ -139,30 +139,7 @@ function Donate() {
             </p>
           </div>
         </section>
-        {/* <section className="d-flex flex-column donate-page-section donate-page-image-section">
-          <h3 className="mt-5 mb-3 section-head">
-            Your Donations Create Impact
-          </h3>
-          <hr className="heading-hr align-self-center" />
-          <img className="middleImg" src={middleImg}></img>
-          <div className="donation-impact-data d-flex justify-content-around my-5">
-            <span>
-              <span className="section-head">160+</span>
-              <p className="">Software Development Jobs</p>
-            </span>
-            <span>
-              <span className="section-head">4000+</span>
-              <p className="">Online Students</p>
-            </span>
```

Comment:
Nesting the block-level `<p>` element inside an inline `<span>` element is not syntactically correct HTML.  (You can validate your HTML at https://validator.w3.org/nu/#textarea , making adjustments for valid React attributes such as `className`.)